### PR TITLE
chore(contributing): update contributing guidelines

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,145 +1,127 @@
...
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
```
Comment (stephaniecoates) [inline CONTRIBUTING.md]: should this just be `$ npm install`?

**Review by JAdshead — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,145 +1,127 @@
...
-* **test** (adding missing tests, refactoring tests; no production code change)
-* **chore** (updating build/env/packages, etc; no production code change)
+- `npm run lint`
 
-#### Scope
+Verifies that your code matches the American Express code style defined in
+[`babel-preset-amex`](https://github.com/americanexpress/babel-preset-amex).
 
-The *scope* describes the affected code. The descriptor may be a route, component, feature, utility, etc. It should be one word or camelCased, if needed:
+- `npm pretest`
```
Comment (stephaniecoates) [inline CONTRIBUTING.md]: I think the bullet points on these commands is funky formatting and hard to read. What about just leaving them as headers w/ no bullet point? Or, if more emphasis is needed, bold them like **`npm pretest`?**

**Review by stephaniecoates — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nellyk) [reply CONTRIBUTING.md]: thank you for catching this!

File: CONTRIBUTING.md (hunk 2 above)
Comment (nellyk) [reply CONTRIBUTING.md]: thank you for catching this!

File: CONTRIBUTING.md (hunk 1 above)
Comment (mtomcal) [reply CONTRIBUTING.md]:
```suggestion
    $ npm install
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (mtomcal) [reply CONTRIBUTING.md]: There's no `pretest` job in this repo.

Comment (nellyk) [reply CONTRIBUTING.md]: updated on f1a9276

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,145 +1,124 @@
...
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
 
-```bash
-fix(home/transactions): txn actions not working
-```
+## Submitting a new feature
 
-This header makes it clear that the fix is limited in scope to transactions within the home route/context.
+When submitting a new feature request or enhancement of an existing features please review the following:-
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
When submitting a new feature request or enhancement of an existing features please review the following:
```

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by anescobar1991 — APPROVED** [review]

**Review by stephaniecoates — APPROVED** [review]

**Pull request merged**